src/infra/workload/releaseunit/modules/stamp/network.tf

```diff
@@ -24,6 +24,13 @@ resource "azurerm_virtual_network" "stamp" {
   location            = azurerm_resource_group.stamp.location
   address_space       = [module.subnet_addrs.base_cidr_block]
 
+  # For production workloads we recommend to enable DDoS protection Standard plan when exposing your AKS ingress on public IPs
+  # You can link this to an existing DDoS protection plan or create a new one
+  # ddos_protection_plan {
+  #   id     = var.ddos_protection_plan_id
```

Comment:
Where is this variable declared and how does one get the value of the procection plan?

Reply:
damn, you dont miss a thing :D I was wondering if I should actually include all that, too? would be all commented out, too. do you want me to?

Reply:
Maybe just explain what needs to be done for this to work? "add this variable to variables.tf, get the plan ID from..."

Reply:
Better :) Now I've learned that it's supposed to be resource ID.